configs/logging/logrotate/forseti

```diff
@@ -0,0 +1,7 @@
+/var/log/forseti.log {
+  daily
+  rotate 30
+  missingok
+  notifempty
+  create 640 root root
```

Comment:
If forseti server isn't running as root it won't be able to write to the log. This should be set to be owned by the user the server runs as.

Reply:
Done